redun/cli.py

```diff
@@ -719,7 +719,11 @@ def import_script(filename_or_module: str, add_cwd: bool = True) -> ModuleType:
             add_import_path(os.getcwd())
         module_name = filename_or_module
 
-    module = importlib.import_module(module_name)
+    try:
+        module = importlib.import_module(module_name)
+    except ModuleNotFoundError as e:
+        logger.error(f"Failed to run workflow from Python script file named {filename_or_module}.")
+        raise
```

Comment:
We can raise the `RedunClientError` in order to get a nicer error message. It also suppresses showing the traceback which is more appropriate for an known error.

```suggestion
    except ModuleNotFoundError:
        raise RedunClientError(f"Cannot find Python script file or module: {filename_or_module}")
```

It looks like this:

```
redun run bad.py main
RedunClientError: Cannot find Python script file or module: bad.py
```